src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -941,7 +804,10 @@ private void OnCanvasClicked(object sender, MouseButtonEventArgs e)
 
         private void OnContextMenuOpened(object sender, RoutedEventArgs e)
         {
-            ViewModel.InCanvasSearchViewModel.SearchText = String.Empty;
+            ViewModel.InCanvasSearchViewModel.SearchText = string.Empty;
+
+            var ctrl = outerCanvas.ContextMenu.ChildOfType<InCanvasSearchControl>();
+            ctrl.SearchTextBox.Focus();
```

Comment:
setting focus here is reliable because `ContextMenu.Opened` event triggers every time when context menu is actually opened